examples/bucketaccessclass.yaml

```
@@ -2,8 +2,8 @@ kind: BucketAccessClass
 apiVersion: objectstorage.k8s.io/v1alpha1
 metadata:
   name: sample-bac
-driverName: ceph.objectstorage.k8s.io
+driverName: ceph-cosi-driver.ceph.objectstorage.k8s.io
```

What if this were just `cosi.ceph.objectstorage.k8s.io`? 'driver' seems like an unnecessary distinction to me, and 'ceph' is already codified in `ceph.objectstorage.k8s.io`. 
 
```suggestion
driverName: cosi.ceph.objectstorage.k8s.io
```